front/app/api/invites/__mocks__/useInvites.ts

```diff
@@ -0,0 +1,116 @@
+export const invitesData = [
```

Comment:
```suggestion
export const invitesData: IInviteData[] = [
```
This will ensure the types are correct. There are a few errors.

(For `Type '"4nce3671h"' is not assignable to type '"string"'.ts(2322)` you can add `as const` after the string value.)